docs/admin/install/steps/install-python.rst

```diff
@@ -6,27 +6,32 @@ Python modules
    We're using virtualenv to install Weblate in a separate environment from your
    system. If you are not familiar with it, check virtualenv :doc:`venv:user_guide`.
 
+.. warning::
+
+   Run the following commands in an empty directory
 
 #. Create the virtualenv for Weblate:
 
    .. code-block:: sh
 
-        virtualenv --python=python3 ~/weblate-env
+        python3 -m venv .
```

Comment:
What is wrong on using `virtualenv`?

Reply:
It doesn't matter in that sense, so you could just as well replace this with mine, it's shorter...

Reply:
I've replaced it with even shorter in 4adc387772162f7d6b1322930644079f4873bd63